generators/server/files.js

```diff
@@ -167,8 +167,8 @@ const couchbaseFiles = {
       path: SERVER_MAIN_SRC_DIR,
       templates: [
         {
-          file: 'package/repository/CustomReactiveCouchbaseRepository.java',
-          renameTo: generator => `${generator.javaDir}repository/CustomReactiveCouchbaseRepository.java`,
+          file: 'package/repository/N1qlCouchbaseRepository.java',
+          renameTo: generator => `${generator.javaDir}repository/ReactiveN1qlCouchbaseRepository.java`,
         },
       ],
     },
```

Comment:
I Agree, but I think that this name is no longer justified, since `CouchbaseRepository` is now using N1QL by default, unlike previous versions. Our new repository is now here to set `ScanConsistency` and add some utility functions to avoid copy/paste. I will call it `JHipsterCouchbaseRepository`. This repository will be probably removed when the issues are fixed in Spring Data Couchbase side, which seems to be soon, as they are already implemented, I'm just waiting for a release.